ci-operator/step-registry/assisted/vsphere/setup/configure/assisted-vsphere-setup-configure-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# ensure LEASED_RESOURCE is set
+if [[ -z "${LEASED_RESOURCE}" ]]; then
+    echo "$(date -u --rfc-3339=seconds) - failed to acquire lease"
+    exit 1
+fi
+
+cloud_where_run="VMC"
```

Comment:
@rvanderp3 thanks for reviewing.
In contrast, the vsphere storage and networking configuration should not be included in the logic. It doesn't make sense to create and approve a PR if we want to change something (such as changing the datastore). 
we can resolve this issue by creating a solution for configuration like config map on kubernetes.

Reply:
One of the benefits of tracking the changes via PR is the visibility of changes and a tracked history.  We have been keeping the configuration in the release repo since we started testing CI on vSphere as a way to manage and update the configuration of jobs.  Yes, I and the rest of the maintainers of the vSphere CI environment understand this repository is public.  Though it never hurts to be reminded of that :) .  As I mentioned above, I'm not opposed to the way you've done this, it's just in contrast to the way other vSphere jobs are composed.

Reply:
thanks @rvanderp3:)

Reply:
no problem at all @michaellevy101 :)